apis/metal3.io/v1alpha1/baremetalhost_types.go

```diff
@@ -119,6 +123,10 @@ const (
 	OperationalStatusError OperationalStatus = "error"
 
 	OperationalStatusDelayed = "delayed"
+
+	// OperationalStatusUnmanaged is the status value when the host is
+	// marked unmanaged via the unmanaged annotation
+	OperationalStatusUnmanaged OperationalStatus = "unmanaged"
```

Comment:
It is slightly confusing that there is also an "Unmanaged" provisioning state that has operational status "discovered". I wonder if there are any other possible names.

Reply:
Yeah I thought the same but wasn't able to come up with an alternative that conveyed the right meaning, suggestions welcome! :)

From your provisioner API suggestions:
* Ignored - This doesn't convey quite the same meaning to me, but it's fairly close if we document that an "ignored" BMH is frozen in its current state, without any management by the provisioner.
* Paused - overlaps with the existing paused annotation which has slightly different behavior   
* Deregistered - this could work as the operational status, but it reflects the internal state (deregistered from the provisioner) not really the state of the BMH resource? 

IMO unmanaged conveys the status most accurately, but if we think it's too confusing having both state and status with that name, I'm thinking "ignored" or perhaps "frozen"

Reply:
I renamed the status and the annotation to "deregistered"